arrows/core/video_input_image_list.cxx

```diff
@@ -362,7 +362,7 @@ ::frame_image()
 {
   if ( !d->m_image && this->good() )
   {
-    LOG_DEBUG( logger(), "reading image from file \"" << *d->m_current_file << "\"" );
+    LOG_INFO( logger(), "reading image from file \"" << *d->m_current_file << "\"" );
```

Comment:
This is somewhat subjective.  I'm not going to argue too strongly one way or the other on whether this message should be `debug` or `info`.  However, given that this is *video* input there are likely a lot of these messages that will be reported, and the average user of video input that wants high-level `info` comments probably doesn't want to see a message about every frame that is loaded.

Note that logging is configurable and can be controlled on a per algorithm basis.  So if Viame wants to see these messages it might be better to setting the log level for `arrows.core.video_input_image_list` to `debug` in the Viame logger configuration.